libs/langchain/langchain/indexes/_sql_record_manager.py

```diff
@@ -59,18 +59,18 @@ class UpsertionRecord(Base):  # type: ignore[valid-type,misc]
     __tablename__ = "upsertion_record"
 
     uuid = Column(
-        String,
+        String(36),
```

Comment:
whats the motivation here?

Reply:
im trying to use mysql dialect, and it need a VARCHAR length value